server/parsec/cli/run.py

```diff
@@ -455,6 +455,17 @@ def run_cmd(
                     app_config=app_config,
                 )
             )
+        # Ignore some noisy cancellation
+        # Note that this is no longer necessary with the latest anyio version (version 4.6)
```

Comment:
I suppose there are breaking changes if we upgrade `anyio`? Have you considered/tried it?

Reply:
There is already an issue for this: https://github.com/Scille/parsec-cloud/issues/7356
I'll link this comment.